README.md

```diff
@@ -114,3 +114,6 @@ Deployment is done by calling the following make command:
 ```
 make deploy
 ```
+
+# Cross referencing documentation
+It's possible to cross reference packages that have been documented with Sphinx within rosindex. Please refer to [this document](https://github.com/ros2/ros2_documentation/blob/master/Inter-Sphinx-Example.rst) to get more information.
```

Comment:
Will this file be added in a pull request after https://github.com/ros2/ros2_documentation/pull/5 ?

Reply:
Now that I think about it, it would have made a total sense to include it in the PR you're pointing, but it was added to a separate one here: [ros2/ros2_documentation#6](https://github.com/ros2/ros2_documentation/pull/6).